lib/checks/navigation/region.js

```diff
@@ -30,16 +30,14 @@ function isLandmark(virtualNode) {
 	var explictRole = (node.getAttribute('role') || '').trim().toLowerCase();
 
 	if (explictRole) {
-		if (explictRole === 'form') {
-			return !!aria.labelVirtual(virtualNode);
-		}
 		return landmarkRoles.includes(explictRole);
 	} else {
 		// Check if the node matches any of the CSS selectors of implicit landmarks
 		return implicitLandmarks.some(implicitSelector => {
 			let matches = axe.utils.matchesSelector(node, implicitSelector);
 			if (node.tagName.toLowerCase() === 'form') {
-				return matches && !!aria.labelVirtual(virtualNode);
+				let title = node.getAttribute('title') || '';
+				return matches && (!!aria.labelVirtual(virtualNode) || !!axe.commons.text.sanitize(title) );
```

Comment:
minor detail, why aim to sanitize title if tile is empty (in some cases)? worth adding an && to check for that.

Reply:
Ok, I pushed another commit to not sanitize empty strings.